preinstall.js

```diff
@@ -25,9 +25,14 @@ if (!conf) {
   console.error(`Platform ${process.platform} is not supported`)
   process.exit(1)
 }
-const endpointURL = `https://s3-eu-west-1.amazonaws.com/ledger-lib-ledger-core/${libcoreVersion}/${
-  conf.dir
-}`
+
+const dir =
+  process.platform === 'linux' && Number(process.version.match(/^v(\d+\.\d+)/)[1]) >= 10
+    ? `${conf.dir}-arch_ssl_1_1`
+    : conf.dir
```

Comment:
do we still keep this switch as we said linux would just switch to the new implementation? @KhalilBellakrid

Reply:
Yes, for the moment it should be seamless from your side